pytorch_lightning/trainer/trainer.py

```diff
@@ -258,6 +258,14 @@ def __init__(
                 :class:`datetime.timedelta`, or a dictionary with keys that will be passed to
                 :class:`datetime.timedelta`.
 
+            move_metrics_to_cpu: Whether to force internal logged metrics to be moved to cpu.
```

Comment:
there are other items that are not alphabetically ordered, e.g., the min/max steps and overfit_batches as far as I can tell

Reply:
yeah, I don't think we shouldn't change min/max steps (user readability and convenience).

Reply:
Actually, there were more 😂 😂